cron_scripts/lib/ZCS/API.pm

```diff
@@ -0,0 +1,363 @@
+package ZCS::API;
```

Comment:
This code should be run through perltidy.

Also, is this code a reduction/minimization/remix of https://github.com/plobbes/zcs-api/blob/master/lib/ZCS/API.pm ? It sure looks like it, but there's no retention or the copyright, nor any mention of the other lib for that matter.

Reply:
Done with perltidy and added head1 section for reference